python-libraries/data-platform-catalogue/data_platform_catalogue/entities.py

```diff
@@ -50,22 +66,39 @@ def from_data_product_metadata_dict(metadata: dict, version, owner_id: str):
             description=metadata["description"],
             version=version,
             owner=owner_id,
+            owner_display_name=metadata["dataProductOwnerDisplayName"],
+            maintainer=metadata.get("dataProductMaintainer"),
+            maintainer_display_name=metadata.get("dataProductMaintainerDisplayName"),
             email=metadata["email"],
+            status=DataProductStatus[metadata["status"]],
             retention_period_in_days=metadata["retentionPeriod"],
             domain=metadata["domain"],
             dpia_required=metadata["dpiaRequired"],
+            dpia_location=metadata.get("dpiaLocation"),
+            last_updated=datetime.strptime(metadata["lastUpdated"], "%Y%m%d"),
```

Comment:
Just to check, have you tested this is the format datahub expects for this field? Just wondering because I encountered another timestamp that expected an integer somewhere.

Also, can you extract the date format string to a constant? e.g. DATAHUB_DATE_FORMAT = '%Y%m%d'

Reply:
These are being added as custom fields and not as part of the datahub model. I think it might come out a bit weird because of the datetimes but it should be accepted as a string like `datetime.datetime(2021, 1, 1, 0, 0)`